modules/nlu/src/backend/engine.ts

```diff
@@ -125,28 +126,23 @@ export default class Engine implements NLUEngine {
     let model: Partial<UntrainedModel> = {
       startedAt: new Date(),
       languageCode: input.languageCode,
+      success: false,
       data: {
         input
       }
     }
 
     try {
       const output = await Trainer(input, Engine.tools)
-      if (!output) {
-        model.success = false
-      } else {
-        _.merge(model, { success: true, data: { output } })
-      }
+      model.data.output = output
+      model.success = true
```

Comment:
model sucess should not be true if output is undefined

Reply:
Fixed in next PR